.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "apoc-core"]
 	path = apoc-core
 	url = https://github.com/neo4j/apoc
-	branch = dev
```

Comment:
This is temporary, was just me getting the submodules to sync, will remove when/if core is merged